cores/esp8266/Esp.h

```diff
@@ -92,7 +92,7 @@ class EspClass {
         void wdtDisable();
         void wdtFeed();
 
-        void deepSleep(uint64_t time_us, RFMode mode = RF_DEFAULT);
+        void deepSleep(uint64_t time_us, RFMode mode = RF_DEFAULT, bool instantly = false);
```

Comment:
Suggest adding new function, `deepSleepInstant`. This way, only one of `system_deep_sleep_instant`/`system_deep_sleep` will be linked into the final application, depending on what the user needs.

Reply:
Ok, done.